app/database/operator/server_data_operator/handlers/post.ts

```diff
@@ -392,14 +428,8 @@ const PostHandler = <TBase extends Constructor<ServerDataOperatorBase>>(supercla
         }
 
         const {firstPost, lastPost} = getPostListEdges(posts);
-
-        // Channel Id for this chain of posts
         const channelId = firstPost.channel_id;
-
-        // Find smallest 'create_at' value in chain
         const earliest = firstPost.create_at;
-
-        // Find highest 'create_at' value in chain; -1 means we are dealing with one item in the posts array
```

Comment:
Some of these comments seem outdated, since we are no longer doing a find (and therefore there is no -1) but we are using the `getPostListEdges` function.